core/CMake_symbol_check.cmake

```diff
@@ -47,8 +66,22 @@ execute_process(COMMAND
   ERROR_VARIABLE readelf_error
   OUTPUT_VARIABLE output
   )
+
+is_i3450_binutils_version(is_i3450)
+
+# XXX i#3450: Check for binutils/readelf version 2.30, and if yes, then ignore errors
+# for 32-bit release build. Binutils bug #24382 has been filed. Both earlier and later
+# versions seem ok.
+if (X86 AND NOT X64)
+  if (NOT DEBUG)
```

Comment:
nit: Can fold into the if above to reduce the nesting

Reply:
I remember a review where we did the opposite because the conditions were/are somewhat unrelated! Changed this one into one liner.